src/state/migrations/2.ts

```diff
@@ -24,6 +24,7 @@ export function migration2(state: PersistedState | undefined): V2State {
     dismissable: {
       hasSeenPredictionMarketIntroDialog: hasSeenPredictionMarketsIntro ?? false,
       dismissedAffiliateBanner: false,
+      hasSeenUnlimitedAnnouncement: false,
```

Comment:
I don't totally understand why we are modifying old migrations. Is it really necessary?